### DISCUSSION TITLE
routes not working when using astro build

### DISCUSSION THREAD

**Discussion by bkyerv** [TanStack/router#422] [Q&A]
I am building a simple app using astro build. After implementing a router mechanism (as shown in the [docs](https://tanstack.com/router/v1/docs/quick-start)) and attempting the 'about' route I am facing an error saying '404: Not found'. I feel that routing implementation of tanstack/router is conflicting with file structure of used by Astro. I am wondering if anyone has come across this issue and has solved it?

**Comment by Moshyfawn** [comment]
Take a look at [this example](https://github.com/TanStack/router/tree/beta/examples/react/with-astro-ssr)
